projects/packages/forms/src/dashboard/components/create-form-button/index.tsx

```diff
@@ -27,7 +27,7 @@ type CreateFormButtonProps = {
  * @return {JSX.Element}                 The button to create a new form.
  */
 export default function CreateFormButton( {
-	label = __( 'Create a free form', 'jetpack-forms' ),
+	label = __( 'Create a form', 'jetpack-forms' ),
```

Comment:
Was about to suggest we remove the prop but I saw in `EmptyResponses` component we use "Create a new form" copy. 👍